runtime/xamarin/runtime.h

```diff
@@ -233,6 +233,7 @@ void			xamarin_process_nsexception_using_mode (NSException *ns_exception, bool t
 void			xamarin_process_managed_exception (MonoObject *exc);
 void			xamarin_process_managed_exception_gchandle (guint32 gchandle);
 void			xamarin_throw_product_exception (int code, const char *message);
+guint32			xamarin_create_product_exception (int code, const char *message);
```

Comment:
minor: mixed tabs/spaces ?

Reply:
No, just 4-vs-8 spaces for tabs (the file is indented to match 4 space, GitHub renders using 8 spaces).

You can see the same thing happening elsewhere in the file if you expand the diff.